src/frontend/src/components/CollapsibleInstructions/index.tsx

```diff
@@ -28,6 +30,7 @@ interface Props {
 
 const CollapsibleInstructions = ({
   buttonSize = "xxxs",
+  downloadTSVOption,
```

Comment:
I feel weird about this. The collapsible instructions component is generic and shouldn't have knowledge of other components or app functionality. At the very least, this should be given a more generic name, maybe something like `additionalHeaderLink`.

Reply:
I think renaming is fine for now